<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainForm</class>
 <widget class="QWidget" name="MainForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Мониторинг действий пользователя на клавиатуре</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Triangular</enum>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab2">
      <attribute name="title">
       <string>Пользователи</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="1" column="0">
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="1">
          <widget class="QComboBox" name="comboUser">
           <property name="enabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="buttonMonitoring">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Включить мониторинг действий</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QGroupBox" name="groupBox_2">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="title">
            <string>Информация о пользователе</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="7" column="1">
             <widget class="QLabel" name="keyDelay">
              <property name="text">
               <string>0</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="8" column="1">
             <widget class="QLabel" name="keySearch">
              <property name="text">
               <string>0</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="editUserName"/>
            </item>
            <item row="9" column="0">
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>Количество нажатий в секунду</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="buttonAdd">
              <property name="text">
               <string>Добавить</string>
              </property>
             </widget>
            </item>
            <item row="7" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Среднее время удержания клавиши</string>
              </property>
             </widget>
            </item>
            <item row="8" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Среднее время поиска клавиши</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Имя пользователя:</string>
              </property>
             </widget>
            </item>
            <item row="6" column="0" colspan="2">
             <widget class="QLabel" name="label_17">
              <property name="text">
               <string>Статистика выбранного пользователя:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <spacer name="verticalSpacer_4">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="11" column="1">
             <widget class="QLabel" name="keyFuncs">
              <property name="text">
               <string>0</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="11" column="0">
             <widget class="QLabel" name="label_15">
              <property name="text">
               <string>Количество функциональных клавиш</string>
              </property>
             </widget>
            </item>
            <item row="10" column="1">
             <widget class="QLabel" name="keyCmbs">
              <property name="text">
               <string>0</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="9" column="1">
             <widget class="QLabel" name="keyNumber">
              <property name="text">
               <string>0</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="10" column="0">
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>Количество сочетаний</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="buttonDelete">
              <property name="text">
               <string>Удалить</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab1">
      <attribute name="title">
       <string>Логгирование нажатий</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="1" colspan="4">
        <widget class="QTextEdit" name="textLogging">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="4">
        <widget class="QPushButton" name="buttonClear">
         <property name="text">
          <string>Очистить журнал</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab3">
      <attribute name="title">
       <string>Распознование</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="3" column="0">
        <widget class="QTextEdit" name="textCmbs">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QTextEdit" name="textShortcuts">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Сочетания клавиш</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Сочетания букв русского языка</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
